packages/cli/src/commands/hydrogen/upgrade.ts

```diff
@@ -999,7 +999,8 @@ export async function displayDevUpgradeNotice({
 
   const isPrerelease = semver.prerelease(currentVersion);
 
-  if (isPrerelease) {
+  if (isPrerelease || /^[a-z]+$/i.test(currentVersion)) {
+    // Skip prereleases or versions like 'next' or 'latest'
```

Comment:
It would be really nice to make it work with the next release. Even if we don't have all the fancy ui with what's changed. Just making it easy to upgrade an existing project to next seems to be a good idea.